parsec/api/data/manifest.py

```diff
@@ -75,7 +75,7 @@ class SCHEMA_CLS(BaseSchema):
         def make_obj(self, data: Dict[str, Any]) -> "WorkspaceEntry":
             return WorkspaceEntry(**data)
 
-    name: str
+    name: EntryName
```

Comment:
Not so sure about the fix of this typing because in the code we use this class with `name` as str in input.